starfish/agent/surfer_agent.py

```diff
@@ -215,7 +215,8 @@ def search_listings(self, text, sort=None, offset=100, page=0):
         # TODO: implement search listing in surfer
         pass
 
-    def purchase_asset(self, listing, account):
+    def purchase_asset(self, listing, account, id=None, status=None,
```

Comment:
I assume the `id` is the purchase?  Codacy complains about the single use of `id`. 
So can this be renamed to `purchase_id` ?